src/traces/parcoords/attributes.js

```diff
@@ -120,36 +119,26 @@ module.exports = {
         description: 'The dimensions (variables) of the parallel coordinates chart. 2..60 dimensions are supported.'
     },
 
-    line: extendFlat({},
-
+    line: extendFlat(
         // the default autocolorscale isn't quite usable for parcoords due to context ambiguity around 0 (grey, off-white)
-        // autocolorscale therefore defaults to false too, to avoid being overridden by the  blue-white-red autocolor palette
+        // autocolorscale therefore defaults to false too, to avoid being overridden by the blue-white-red autocolor palette
         extendDeep(
-            {},
```

Comment:
Why did you change this?

`colorAttributes` and friends are used in other place in plotly.js, so we can't _just_ extend them here. We need to make a copy of them and then overwrite a few keys.

Reply:
@etpinard `colorAttributes(str)` returns a new object every call, no need to copy a copy.